content/utils/experimenterApi.js

```diff
@@ -5,6 +5,12 @@ export default class ExperimenterAPI extends API {
     return new URL(`api/v${version}/`, this.environment.experimenterUrl).href;
   }
 
+  async request(options) {
```

Comment:
This seems like a no-op. Couldn't you remove it?

Reply:
tslint wasn't happy with me  when i had searchParams for https://github.com/mozilla-extensions/normandy-devtools/pull/499/files#diff-6d30f20f347f56d86c0c9ef1029997e54bb2713522f3dced5efb6ba0b90f13c7R29

wasn't sure if tehre was another way to make it happy

Reply:
The super method has JSDoc-style type annotations that don't mention `data`, so it makes sense that tslint was grumpy. Adding this interstitial method erases the type information, so it just turns off the checks. Better would be to add `data` as a possible parameter here:

https://github.com/mozilla-extensions/normandy-devtools/blob/ee192ff3d9b5786e766905921ee7b9d348ffecd2/content/utils/api.js#L19-L27

Without converting it over to Typescript (which would be better), you could probably just add

```js
 * @param {Object} [args.data] Data to convert JSON serialize into the querystring (for GET and HEAD requests) or the request body (for other methods).
```